app/export.go

```diff
@@ -414,7 +413,7 @@ func (a *App) BuildPostReactions(postId string) (*[]ReactionImportData, *model.A
 		user, err = a.Srv.Store.User().Get(reaction.UserId)
 		if err != nil {
 			if err.Id == store.MISSING_ACCOUNT_ERROR { // this is a valid case, the user that reacted might've been deleted by now
-				mlog.Info(fmt.Sprintf("Skipping reactions by user %v, since the entity doesn't exist anymore", reaction.UserId))
+				mlog.Info("Skipping reactions by user since the entity doesn't exist anymore", mlog.String("reaction_UserId", reaction.UserId))
```

Comment:
Seeing that there are multiple conventions we've been using for naming variables like this, but looks like snake_case seems to be the most common so I'd suggest to just use `user_id` here.
```suggestion
				mlog.Info("Skipping reactions by user since the entity doesn't exist anymore", mlog.String("reaction_UserId", reaction.UserId))
```

Reply:
Aha I see now, I've made the change now @ali-farooq0 :+1: